.github/workflows/run_tests_win_mingw.yml

```diff
@@ -46,9 +48,24 @@ jobs:
         run: cat libnetcdf.settings
 
       - name: (Autotools) Build Library and Utilities
-        run: make -j 8 LDFLAGS="-no-undefined -Wl,--export-all-symbols"
```

Comment:
Why remove the -no-undefined?

Reply:
It's already in the `Makefile.am`, and I like having the flags necessary for things to build actually specified by the build system.

I tried removing `-Wl,--export-all-symbols` as well, but that led to a number of "unresolved reference to symbol nc..." errors at link time.  I might be able to make more progress by specifying `AM_CPPFLAGS=-DDLL_EXPORT -DDLL_NETCDF` in the top-level `Makefile.am` so the source files properly specify `__declspec(dllexport)`, but given that CMake fails to link I'm not sure that would be enough.